tests/Finder/TestFrameworkFinderTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Copyright © 2017-2018 Maks Rafalko
+ *
+ * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Infection\Tests\Finder;
+
+use Infection\Finder\Exception\FinderException;
+use Infection\Finder\SourceFilesFinder;
+use Infection\Finder\TestFrameworkFinder;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Finder\Finder;
+
+class TestFrameworkFinderTest extends TestCase
+{
+    public function test_it_can_load_a_custom_path(): void
```

Comment:
Please remove the `: void`  since that breaks test on 7.0

Reply:
ah of course, habit :)